### PR TITLE
build: make fd/pandoc optional

### PR DESCRIPTION
**Pull request by CameronNemo**

### CODE REVIEW

File: res/build.sh [hunk 1]
```diff
@@ -1,6 +1,7 @@
 #!/bin/sh
 # uses PREFIX from environment
```
Comment (ericonr) [inline res/build.sh:2]: Update the comment.

File: res/build.sh [hunk 2]
```diff
@@ -1,6 +1,7 @@
...
 : "${PREFIX:=/usr/local}"
+: "${VDOCS_BUILD_MANPAGES:=1}"
```
Comment (ericonr) [inline res/build.sh:4]: maybe just `BUILD_MANPAGES`?

Comment (CameronNemo) [reply res/build.sh]: done

File: res/build.sh (hunk 1 above)
Comment (CameronNemo) [reply res/build.sh:2]: to what?

Comment (ericonr) [reply res/build.sh:2]: `uses PREFIX and BUILD_MANPAGES from environment`

Comment (CameronNemo) [reply res/build.sh]: done

**Pull request merged**